Assets/Scripts/MedicalCanisterPickup.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UnityEngine;
+
+public class MedicalCanisterPickup : MonoBehaviour
+{
+
+    public GameObject healAura;
+
+    private void OnTriggerEnter(Collider other)
+    {
+        if (!other.CompareTag("Player")) return;
+        other.GetComponentInChildren<GrappleGunBehavior>().StopGrapple();
```

Comment:
Instead of dynamically getting this component, it might be worth setting up this reference on start.

Reply:
Done